lib/ControlledVocab/ControlledVocab.js

```diff
@@ -32,58 +32,58 @@ class ControlledVocab extends React.Component {
   });
 
   static propTypes = {
-    stripes: PropTypes.shape({
-      connect: PropTypes.func.isRequired,
-      hasPerm: PropTypes.func.isRequired,
-      formatDate: PropTypes.func.isRequired,
-      intl: PropTypes.shape({
-        formatMessage: PropTypes.func.isRequired,
-      }).isRequired,
-    }).isRequired,
+    actionProps: PropTypes.object,
+    actionSuppressor: PropTypes.object,
+    baseUrl: PropTypes.string.isRequired,
+    columnMapping: PropTypes.object,
+    formatter: PropTypes.object, // eslint-disable-line react/no-unused-prop-types
+    hiddenFields: PropTypes.arrayOf(PropTypes.string),
+    itemTemplate: PropTypes.object,
     label: PropTypes.string.isRequired,
     labelSingular: PropTypes.string.isRequired,
-    objectLabel: PropTypes.string.isRequired,
-    baseUrl: PropTypes.string.isRequired, // eslint-disable-line react/no-unused-prop-types
-    records: PropTypes.string.isRequired,
-    resources: PropTypes.shape({
-      values: PropTypes.shape({
-        records: PropTypes.arrayOf(PropTypes.object),
-      }),
-    }).isRequired,
+    listSuppressor: PropTypes.func,
+    listSuppressorText: PropTypes.string,
     mutator: PropTypes.shape({
       activeRecord: PropTypes.shape({
         update: PropTypes.func,
       }),
       values: PropTypes.shape({
-        reset: PropTypes.func,
+        DELETE: PropTypes.func,
         GET: PropTypes.func,
         POST: PropTypes.func,
         PUT: PropTypes.func,
-        DELETE: PropTypes.func,
+        reset: PropTypes.func,
       }),
     }).isRequired,
-    visibleFields: PropTypes.arrayOf(PropTypes.string),
-    hiddenFields: PropTypes.arrayOf(PropTypes.string),
+    nameKey: PropTypes.string,
+    objectLabel: PropTypes.string.isRequired, // eslint-disable-line react/no-unused-prop-types
+    preCreateHook: PropTypes.func,
+    preUpdateHook: PropTypes.func,
     readOnlyFields: PropTypes.arrayOf(PropTypes.string),
-    columnMapping: PropTypes.object,
-    itemTemplate: PropTypes.object,
-    nameKey: PropTypes.string, // eslint-disable-line react/no-unused-prop-types
-    formatter: PropTypes.object,
-    actionSuppressor: PropTypes.object,
-    actionProps: PropTypes.object,
-    listSuppressor: PropTypes.func,
-    listSuppressorText: PropTypes.string,
+    records: PropTypes.string.isRequired,
+    resources: PropTypes.shape({
+      values: PropTypes.shape({
+        records: PropTypes.arrayOf(PropTypes.object),
+      }),
+    }).isRequired,
     rowFilter: PropTypes.element,
     rowFilterFunction: PropTypes.func,
-    preCreateHook: PropTypes.func,
-    preUpdateHook: PropTypes.func,
+    stripes: PropTypes.shape({
+      connect: PropTypes.func.isRequired,
+      formatDate: PropTypes.func.isRequired,
+      hasPerm: PropTypes.func.isRequired,
+      intl: PropTypes.shape({
+        formatMessage: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+    validate: PropTypes.func,
```

Comment:
Unfrozen caveman programmer say, it's pedantic to comment about comment placement given the sad state of comments in our codebase, but I expect block comments to come before the code they describe as it was originally:
```
/* stuff about validate */
validate: PropTypes.func, 
```
This arrangement makes the block appear to refer to `visibleFields`. I would personally prefer a blank line above the comment and below the line it refers to in order to remove all doubt, but ESLint may disagree. Personally, I find our rules on whitespace suffocating, but I like four character tabs and closing braces on empty lines and `-Wall` and `-pedantic` because I'm a caveman; it's just the way I think.